connectors/src/api/webhooks/webhook_intercom.ts

```diff
@@ -221,19 +221,20 @@ const _webhookIntercomUninstallAPIHandler = async (
   await syncFailed(connector.id, "oauth_token_revoked");
 
   // Attempt to delete the nango connection Id. But fail silently if it fails.
-  nangoDeleteConnection(
+  const nangoRes = await nangoDeleteConnection(
     connector.connectionId,
     NANGO_INTERCOM_CONNECTOR_ID
-  ).catch((e) => {
+  );
```

Comment:
@PopDaph let's avoid .catch at all cost we were not awaiting this and we were not getting the error properly since this function is an async returning a Result